### PR TITLE
(DOCS) Add a note re. where to add the certname

### PR DESCRIPTION
**Pull request by clairecadman**
https://tickets.puppetlabs.com/browse/DOCUMENT-1095

### CODE REVIEW

**Comment by joshcooper** [comment]
Travis is complaining because the git commit summary is missing `(DOCS)`

**Comment by clairecadman** [comment]
@joshcooper I think I already have (DOCS) in the git commit summary?

**Comment by mwaggett** [comment]
@clairecadman it's in the PR summary, but not in the commit message itself (which you can see if you click on the 'Commits' tab from this PR).

**Comment by joshcooper** [comment]
@clairecadman looks like the git commit summary needs updating

File: lib/puppet/defaults.rb [hunk 1]
```diff
@@ -823,6 +823,8 @@ def self.initialize_default_settings!(settings)
           it should match `/\A[a-z0-9._-]+\Z/`.)
         * The special value `ca` is reserved, and can't be used as the certname
           for a normal node.
+          
+          **Note:** For full functionality of the `puppet config print` command and other dependent commands, include the certname in the main section of the puppet.conf file.
```
Comment (joshcooper) [inline lib/puppet/defaults.rb:827]: It almost sounds like `puppet config print` is the most important thing, but it's more that all of puppet assumes certname is set in the main section. Maybe something like "certname" must be set in the "main" section. Setting it in a different section will cause errors.

**Pull request merged**